scriptmodules/emulators/dosbox-staging.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="dosbox-staging"
+rp_module_desc="DOS emulator (enhanced and modern DOSBox fork)"
+rp_module_help="ROM Extensions: .bat .com .exe .sh .conf\n\nCopy your DOS games to $romdir/pc"
+rp_module_licence="GPL2 https://github.com/dosbox-staging/dosbox-staging/blob/master/COPYING"
+rp_module_repo="git https://github.com/dosbox-staging/dosbox-staging.git master"
+rp_module_section="exp"
+
+function depends_dosbox-staging() {
+    getDepends ccache build-essential meson ninja-build libasound2-dev libpng-dev libsdl2-dev libsdl2-net-dev libopusfile-dev libfluidsynth-dev
+}
+
+function sources_dosbox-staging() {
+    gitPullOrClone
+}
+
+function build_dosbox-staging() {
+    local params=(-Dbuildtype=release -Ddefault_library=static)
+
+    # Fluidsynth
+    cd $md_build/contrib/static-fluidsynth
+    make -j $(nproc)
+    export PKG_CONFIG_PATH="${md_build}/contrib/static-fluidsynth/fluidsynth/build"
+
+    cd $md_build
+    meson setup "${params[@]}" build
+    ninja -C build
+
+    md_ret_require=(
+        "$md_build/build/dosbox"
+    )
+}
+
+function install_dosbox-staging() {
+    md_ret_files=(
+        'build/dosbox'
+        'COPYING'
+        'README'
+    )
+}
+
+function configure_dosbox-staging() {
+    local launcher_name="+Start DOSBox-staging.sh"
+    local def="0"
+
+    mkRomDir "pc"
+    rm -f "$romdir/pc/$launcher_name"
+    if [[ "$md_mode" == "install" ]]; then
+        cat > "$romdir/pc/$launcher_name" << _EOF_
+#!/bin/bash
```

Comment:
Better use
```
#!/usr/bin/env bash

Reply:
Done. Though it's not consistent with the other scriptmodule ([here](https://github.com/RetroPie/RetroPie-Setup/blob/90173a8181354fa6bf99da85414b25e63635c241/scriptmodules/emulators/dosbox.sh#L82)).